### PR TITLE
Update Danish Translation (76% / 644)

### PR DESCRIPTION
**Pull request by Stisen1**
- Focused translations on things that have to do with artifacts.
- Focused translations on magic prompts and descriptions.

### CODE REVIEW

File: files/lang/dk.po [hunk 1]
```diff
@@ -9488,17 +9563,21 @@ msgid ""
...
 msgid ""
 "You spy a gleaming object poking up out of the ground. You send a member of "
 "your party over to investigate. He comes back with a golden helmet in his "
 "hands. You realize that it must be the helmet of the legendary Anduran, the "
 "only man who was known to wear solid gold armor."
 msgstr ""
+"Du får øje på en skinnende genstand, der stikker op af jorden. Du sender et "
+"medlem af din hær over, for at undersøge sagen. Han kommer tilbage med en "
+"gylden hjelm i hænderne. Du indser, at det må være den legendariske Andurans "
+"hjelm, den eneste mand, der var kendt for at bære solidt guldudstyr."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:9577]: You're right! How did I miss this!

File: files/lang/dk.po [hunk 2]
```diff
@@ -9684,35 +9780,39 @@ msgid ""
 "Removes all negative spells cast upon your forces, and restores up to "
 "%{count} HP per level of spell power, per creature."
 msgstr ""
+"Fjerner alle negative besværgelser, som din hær er blevet underlagt, og "
+"genopretter op til %{count} HP pr. magikraftspoint, pr. væsen."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:9784]: We do it in both ways. I'll change this "pr." into "per" and then make all other "pr."s into "per", just to keep some consistency :)

File: files/lang/dk.po [hunk 3]
```diff
@@ -2391,9 +2391,8 @@ msgstr "Vis helt"
 msgid "The above spells are available here."
 msgstr "Med nok visdom kan alle disse besværgelser læres af dine helte."
 
-#, fuzzy
 msgid "The spells the hero can learn have been added to their book."
-msgstr "Besværgelser som ses her, er blevet tilføjet din magibog."
+msgstr "De besværgelser, som den besøgende helt kan lære, er blevet tilføjet din magibog."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:2395]: I've shortened it down to 63 symbols.

File: files/lang/dk.po [hunk 4]
```diff
@@ -9919,6 +10034,9 @@ msgid ""
 "This illusionary unit does the same damages as the original, but will vanish "
 "if it takes any damage."
 msgstr ""
+"Skaber en illusionær trop, der dublerer en af dine eksisterende tropper. "
+"Denne illusionære trop giver det samme i skade som originalen, men den "
+"forsvinder, hvis den tager skade!"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:10039]: Sigh... It has been changed to a dot.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Stisen1 , thanks a lot for the update!

**Pull request merged**